framework/deproxy_auto_parser.py

```diff
@@ -163,6 +163,8 @@ def __prepare_expected_response_for_request(
 
         is_cache = "age" in received_response.headers
         if http2 or is_cache:
+            # Tempesta doesn't cache "set-cookie" header
+            expected_response.headers.delete_all("set-cookie")
             self.__prepare_chunked_expected_response(expected_response)
```

Comment:
```suggestion
        if is_cache:
            # Tempesta doesn't cache "set-cookie" header
            expected_response.headers.delete_all("set-cookie")
        if http2 or is_cache:
            self.__prepare_chunked_expected_response(expected_response)
```
TempestaFW removes `set-cookie` header for a cache response. but not for a http2 response.